.github/workflows/MediaConch_Checks.yml

```diff
@@ -0,0 +1,66 @@
+name: Checks
+
+on: [push, pull_request]
+
+jobs:
+  unix:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Dependencies
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            sudo apt-get install -y zlib1g-dev libxml2-dev libxslt-dev libcurl4-gnutls-dev libsqlite3-dev libjansson-dev libevent-dev libxml2-utils qtbase5-dev qtwebengine5-dev qt5-qmake qt5-default xmlstarlet ffmpeg sshpass
+            sudo curl -L http://stahlworks.com/dev/sfk/sfk-linux-64.exe -o /usr/local/bin/sfk
+            sudo chmod +x /usr/local/bin/sfk
+          fi
+          if [ "$RUNNER_OS" == "macOS" ]; then
+            brew install automake libxslt sqlite jansson libevent qt sfk xmlstarlet ffmpeg
+          fi
+          sudo pip install git+https://github.com/vi/mkvparse.git
+          git -C .. clone --depth=1 https://github.com/MediaArea/ZenLib.git

```

Comment:
Note for long term (not for this PR): we could build MediaInfoLib with less features (we don't use all formats in tests) or use a build from MediaInfoLib (and ZenLib) GitHub actions when they are made.